### PR TITLE
javascript-javascript2-week2

### PR DESCRIPTION
**Pull request by suvarna-ratna**

### CODE REVIEW

File: javascript/javascript2/week2/hyf-bay/main.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+const userInput = document.getElementById("inputName");
+const userOption = document.getElementById("selectOption");
+const userCountry = document.getElementById("countryOption");
+const userPriceRange = document.getElementById("start");
+
+// userInput.addEventListener("change", (event) => {
+//     const searchedProduct = event.target.value.toLowerCase()
+//     return searchedProduct ? renderProducts(products.filter(a=>a.name.toLowerCase().includes(searchedProduct))) :
+//     renderProducts(products)
+// })
```
Comment (jcg-codes) [inline javascript/javascript2/week2/hyf-bay/main.js:40]: Great job, but remember to clean your code like this code you comment, if you don't use it you can remove it.

File: javascript/javascript2/week2/hyf-bay/main.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    : renderProducts(products.sort((a, b) => (a.name > b.name && 1) || -1));
+});
+userCountry.addEventListener("change", () => {
+  const searchedCountry = document.getElementById("countryOption").value;
+  switch (searchedCountry) {
+    case "all":
+      renderProducts(products);
+      break;
+    case "denmark":
+      renderProducts(products.filter((a) => a.shipsTo.includes("Denmark")));
```
Comment (jcg-codes) [inline javascript/javascript2/week2/hyf-bay/main.js]: Try to make your code readable, by using a meaningful word instead like `a`  you can use `product` something you will remember after some years. It's also good when you working with other people.

File: javascript/javascript2/week2/hyf-bay/main.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+      renderProducts(products.filter((a) => a.shipsTo.includes("England")));
+      break;
+  }
+});
+userPriceRange.addEventListener("change", () => {
+  const searchedPrice = document.getElementById("start").value;
+  return searchedPrice
+    ? renderProducts(products.filter((a) => a.price < searchedPrice * 1000))
+    : renderProducts(products);
+});
```
Comment (jcg-codes) [inline javascript/javascript2/week2/hyf-bay/main.js:104]: Well done

**Comment by jcg-codes** [comment]
I can't find other exercises, I don't know if you manage to work on movies exercises and Doubling of number.

File: javascript/javascript2/week2/hyf-bay/main.js (hunk 1 above)
Comment (suvarna-ratna) [reply javascript/javascript2/week2/hyf-bay/main.js:40]: OK THANK YOU FOR FEEDBACK

File: javascript/javascript2/week2/hyf-bay/main.js (hunk 2 above)
Comment (suvarna-ratna) [reply javascript/javascript2/week2/hyf-bay/main.js]: OK done

**Pull request merged**